src/senaite/app/listing/ajax.py

```diff
@@ -39,13 +39,26 @@
 from senaite.core.decorators import readonly_transaction
 from senaite.core.interfaces import IDataManager
 from senaite.core.p3compat import cmp
+from senaite.core.registry import get_registry_record
 from zope import event
 from zope.component import getMultiAdapter
 from zope.component import queryAdapter
 from zope.interface import implementer
 from zope.lifecycleevent import modified
 from zope.publisher.interfaces import IPublishTraverse
 
+NO_AJAX_TRANSITION_TYPES = [
+    "Samples",  # all transition buttons are redirects
+    "AnalysisServices",  # copy action not working in Ajax mode
+    "WorksheetFolder",  # delete action not working in Ajax mode
+    "Client",  # All WF actions redirect back to /cllient-id/analysisrequests
```

Comment:
Although it hurts a lot, I would add "AnalysisRequest" here, specially because there are some sample transitions that happen automatically when all analyses are transitioned to a given status. https://github.com/senaite/senaite.core/pull/2137 would solve the problem with the wf action menu, but the header table with details, as well as sections (e.g. Interpretation results) need to be updated as well because they visibility/mode rely on permissions that in turn depend on the current state of the object. However, this really hurts, because it was really cool to see the transitions to happen automatically in analyses listing from inside sample without refresh :cry:

Reply:
I don't think so. We just need another event handler in core to refresh the header table like we did with the WF menu:
https://github.com/senaite/senaite.core/pull/2137/files#diff-49c8a930865b6a41cb0c62c3b8311f978291057b1eaee39961440e7d58ca2f67R82

Reply:
The commit 71da665eb6a777aa8be2802e37c42b32eb0f38b2 moves the blacklisted types/views into the registry:

<img width="1687" src="https://user-images.githubusercontent.com/713193/191846571-a129434b-4dc0-4a2b-9594-d83d335b2f73.png">

Besides this, I'm going to add another PR to reload the whole page when we are on a sample and all items are in verified state (folderitems are passed here: https://github.com/senaite/senaite.app.listing/pull/87/files#diff-b4197223a6b6c276a83366fa5459557eef9b2467693c7cf03d0ed470719452daR860)

Reply:
> The commit [71da665](https://github.com/senaite/senaite.app.listing/commit/71da665eb6a777aa8be2802e37c42b32eb0f38b2) moves the blacklisted types/views into the registry:
> [...]
> Besides this, I'm going to add another PR to reload the whole page when we are on a sample and all items are in verified state (folderitems are passed here: https://github.com/senaite/senaite.app.listing/pull/87/files#diff-b4197223a6b6c276a83366fa5459557eef9b2467693c7cf03d0ed470719452daR860)

Excellent, thanks!